_uw-research-computing/file-avail-largedata.md

```diff
@@ -285,8 +250,7 @@ log = myprogram.log
 output = $(Cluster).out
 error = $(Cluster).err
 
-## Do NOT list the large data files here
-transfer_input_files = myprogram
+transfer_input_files = osdf://chtc/staging/username/myprogram, file:///staging/username/largedata.tar.gz
 
 # IMPORTANT! Require execute servers that can access /staging
 Requirements = (Target.HasCHTCStaging == true)
```

Comment:
eventually, we'll want to get rid of this.

Reply:
Get rid of what, exactly?

Reply:
Christina is referring to the Requirements line

Reply:
the staging requirement. can leave it for now!